### PR TITLE
adds Orion tests for Crystal Lang

### PR DESCRIPTION
**Pull request by CaDs**
This is the [Orion](https://github.com/obsidian/orion) test of the Framework Benchmarks.
Crystal is a new language that closely resembles Ruby with a goal of removing typed variables and parameters (instead inferencing), whilst maintaining top speed through bindings into C.

Orion is a powerful, simple, rails-esque routing library for HTTP::Server

### CODE REVIEW

File: frameworks/Crystal/orion/benchmark_config.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      "default": {
+        "json_url": "/json",
+        "plaintext_url": "/plaintext",
+        "db_url": "/db",
+        "query_url": "/queries?queries=",
+        "fortune_url": "/fortunes",
+        "update_url": "/updates?queries=",
+        "port": 8080,
+        "approach": "Realistic",
+        "classification": "Fullstack",
```
Comment (joanhey) [inline frameworks/Crystal/orion/benchmark_config.json:14]: Classification will be Micro.
A routing lib isn't a fullstack framework.

Comment (CaDs) [reply frameworks/Crystal/orion/benchmark_config.json]: Fixed.
Thanks for letting me know.

Comment (joanhey) [reply frameworks/Crystal/orion/benchmark_config.json]: De nada :+1:

**Comment by CaDs** [comment]
Looks like Travis is stuck in Pending, but when I check the status on Details the build is green.
Is there anything else I need to do with this PR?

**Pull request merged**